ocaml/idl/datamodel_common.ml

```diff
@@ -8,7 +8,7 @@ open Datamodel_roles
               When introducing a new release, bump the schema minor version to the next hundred
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
-let schema_minor_vsn = 200
+let schema_minor_vsn = 201
```

Comment:
I assume this will be updated later on?

Reply:
I think we should update it everytime we make a change to the DB fields, otherwise its hard to test out the changes starting from a previous build and swapping out xapi binary.